packages/angular_devkit/build_angular/src/builders/browser/schema.json

```diff
@@ -201,7 +201,8 @@
     },
     "deployUrl": {
       "type": "string",
-      "description": "URL where files will be deployed."
+      "description": "URL where files will be deployed.",
+      "x-deprecated": "Use \"baseHref\" option, \"APP_BASE_HREF\" DI token or a combination of both instead."
```

Comment:
Maybe add a link to this doc section as well: https://angular.io/guide/deployment#the-deploy-url

Reply:
Added